src/block/tabs/deprecated.js

```diff
@@ -5,6 +5,16 @@ import { attributes } from './schema'
 import { withVersion } from '~stackable/higher-order'
 
 const deprecated = [
+	{
+		attributes: attributes( '3.12.3' ),
+		migrate: attributes => {
+			return {
+				...attributes,
+				equalTabHeight: true,
+			}
+		},
```

Comment:
As discussed, you will need to include the things in the older **migrate** method for 3.11.9:

```js
let newAttributes = { ...attributes }
newAttributes = deprecateContainerBackgroundColorOpacity.migrate( newAttributes )
newAttributes = deprecateBlockBackgroundColorOpacity.migrate( newAttributes )
return newAttributes
```

This is because deprecation only stops when it encounters the first satisfied deprecation, so all users with an old tabs block will never get migration to the new combined opacity and color.